egs/icsi/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
+
+# Pawel: this one starts from the best tdnn model for ami, those are fairly similar
+# datasets so I would expect similar modelling tricks will work for both
+
+set -e -o pipefail
+# First the options that are passed through to run_ivector_common.sh
+# (some of which are also used in this script directly).
+stage=0
+mic=ihm
+nj=30
+min_seg_len=1.55
+use_ihm_ali=false
+train_set=train
+gmm=tri3  # the gmm for the target data
+ihm_gmm=tri3  # the gmm for the IHM system (if --use-ihm-ali true).
+num_threads_ubm=3
+ivector_transform_type=pca
```

Comment:
I think it is better to make the pca version as default and remove it as an option.